shell/packages/sandstorm-db/db.js

```diff
@@ -112,6 +112,7 @@ DevApps = new Mongo.Collection("devapps");
 //
 // Each contains:
 //   _id:  The application ID string (as with Packages.appId).
+//   appId: The same as _id, included to mimic Packages.appId.
```

Comment:
How would you feel about renaming the collection to DevPackages instead, to hint more at the analogous structure?